python/ql/test/experimental/dataflow/tainttracking/customSanitizer/TestTaint.ql

```diff
@@ -0,0 +1,29 @@
+import experimental.dataflow.tainttracking.TestTaintLib
+
+class CustomSanitizerOverrides extends TestTaintTrackingConfiguration {
+  override predicate isSanitizer(DataFlow::Node node) {
+    exists(Call call |
+      call.getFunc().(Name).getId() = "emulated_authentication_check" and
+      call.getArg(0) = node.asExpr()
+    )
+    or
+    node.asExpr().(Call).getFunc().(Name).getId() = "emulated_escaping"
+  }
+
+  override predicate isSanitizerGuard(DataFlow::BarrierGuard guard) {
+    // exists(Call call |
+    //   call.getFunc().(Name).getId() = "emulated_is_safe" and
+    // )
```

Comment:
Future work?

Reply:
Yep. That was obvious to _me_, but I see how it might not be to others. I added a comment explaining it now :+1: